### PR TITLE
add tabs styles

### PR DESCRIPTION
**Pull request by jim-benson**
Task: task-[718670](https://dev.azure.com/ceapex/Engineering/_sprints/taskboard/Assessments/Engineering/2022-10-19?workitem=718670)

Link: preview-[480](https://design.learn.microsoft.com/pulls/480/components/tabs.html)

Add Tab styles to atlas. Updated to follow Learn rebrand and Assessment update

## Testing

1. Step one
2. Step two
   Expected result:

## Additional information

[Optional]

### CODE REVIEW

**Comment by wibjorn** [comment]
@jim-benson is there a figma for this? also, think we'll need to handle some scrolling on mobile screens

File: js/src/behaviors/snap-scroll.ts [hunk 1]
```diff
@@ -86,7 +86,7 @@ function initSnapScrollClickListeners() {
 
 		updatePaginationState(parentElement, target);
 
-		slide.scrollIntoView({ behavior: 'auto', block: 'nearest', inline: 'start' });
+		slide.scrollIntoView({ behavior: 'auto', block: 'nearest', inline: 'center' });
```
Comment (wibjorn) [inline js/src/behaviors/snap-scroll.ts:90]: unelated change? separate PR?

File: js/src/behaviors/tabs.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+export function initTabs(container: HTMLElement, callback = () => {}) {
+	initTabControlClickListeners(container, callback);
+}
+
+function initTabControlClickListeners(container: HTMLElement, callback: () => void) {
+	if (!container) {
+		return;
+	}
+
+	const buttons = Array.from(container.querySelectorAll('.tab-control')) as HTMLButtonElement[];
```
Comment (wibjorn) [inline js/src/behaviors/tabs.ts:15]: need to use the window.addEventListener pattern instead. Can't assume tabs have all been rendered at page load.

File: css/src/components/tabs.scss [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+		align-items: center;
+		justify-content: center;
+		border: none;
+		background-color: transparent;
+		color: $tabs-link-color;
+		padding-block: 1rem;
+		cursor: pointer;
+
+		&:hover {
+			border-radius: $tabs-link-radius;
```
Comment (wibjorn) [inline css/src/components/tabs.scss:36]: don't think we want radius set inside hover or active states, leads to strange focus circle changes on tab selection

**Comment by wibjorn** [comment]
requires a changeset too

File: css/src/components/tabs.scss (hunk 3 above)
Comment (wibjorn) [reply css/src/components/tabs.scss:36]: also, dragging a mouse through the space between buttons yields some very erratic hover behavior

**Comment by jim-benson** [comment]
@wibjorn not sure if there was a design figma, it was just from https://www.figma.com/file/dtRQlu0reG0a771zrPcaP9/Assessments-Homepage?node-id=3445%3A48664. Though as I was doing this, it brought up some questions. Like the current tabs has two styles and do we need to bring these in to altas as well or not. As well as any other requirements

**Comment by jim-benson** [comment]
> @jim-benson is there a figma for this? also, think we'll need to handle some scrolling on mobile screens. Edit: don't see it in the Atlas Figma, this needs to go through design systems review: https://www.figma.com/file/uVA2amRR71yJZ0GS6RI6zL/%F0%9F%8C%9E-Atlas-Design-Library?node-id=0%3A1

I will wait till I find out if this was reviewed or not

**Pull request closed without merging**